packages/server/src/api/controllers/row/views.ts

```diff
@@ -58,8 +58,9 @@ export async function searchView(
         })
       })
     } else {
+      const operator = query.allOr ? "$or" : "$and"
       query = {
-        $and: {
+        [operator]: {
```

Comment:
This should always be an $and. We need to get sure we AND the view filters with the user filters. Otherwise a user can bypass view filters

Reply:
Discussed on call - removing this change.